2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,77 @@
   You should write and log at least 5 recipes
 */
 
+let favouriteRecipe1 = {};
+
```

Comment:
This code looks correct to me 👍 
One small comment - usually if you're defining an object and you already know the properties and values, it can be easier (to both write and read) to include all the information in your object definition. Using your example, this will look like:
```javascript
let favouriteRecipe1 = {
    title: "Mole",
    servings: 2,
    ingredients: ["cinnamon", "cumin", "cocoa"]
};
```